pyedflib/edfwriter.py

```diff
@@ -817,14 +817,14 @@ def writeSamples(self, data_list, digital = False):
             del dataRecord
             dataRecord = np.array([], dtype=np.int32 if digital else None)
             for i in np.arange(len(data_list)):
-                dataRecord = np.append(dataRecord,data_list[i].ravel()[int(ind[i]):int(ind[i]+sampleFrequencies[i])])
+                dataRecord = np.append(dataRecord, data_list[i][int(ind[i]):int(ind[i]+sampleFrequencies[i])])
```

Comment:
now you removed the `ravel`, was this intentional? right now the fix would onyl apply to the highlevel-interface (which is okay I guess)